### ISSUE TITLE
Resubmit app to Google play

### ISSUE THREAD

**Issue by yairm210** [yairm210/UnCiv#393]
Google disabled the app because of impersonation :/
I'll need to resubmit the app as a different app, which is a massive pain, esp. because of the loss of downloads and reviews :(

**Comment by yairm210** [comment]
I submitted an appeal, should get an answer within three days, I think there's how that we can keep the same app if we just change the icon

**Comment by yairm210** [comment]
I'm pretty sure it's because the icon is similar to Civ, so I told them in the appeal what the game is and that I would change the icon

**Comment by yairm210** [comment]
...was denied.

I'll have to open a new app. It'll have the same name, but everything else is wiped clean. Very frustrating.

**Comment by ninjatao** [comment]
You still have support from all your fans.

**Comment by reijin90** [comment]
Sad to hear it got denied... I'm pretty sure a different icon and a copyright notice about civ 5 being the respective owners IP will make it safe for the resubmission. Best would be to not mention civ 5 at all though. I'm pretty sure you will get enough exposure through word of mouth. :)

On a side note: how can I donate?

**Comment by reijin90** [comment]
@yairm210 fair point. Regarding servers: you could make them open source and the community could host.

Yes there are many others! Biggest ones are probably:
Amazon 
F-Droid
